### PR TITLE
someone asked i code holoparasites playing music so here we are???

### PR DESCRIPTION
**Pull request by silicons**
## Why It's Good For The Game

@zeroisthebiggay wants to play jojo music

## Changelog
:cl: silicons
add: holoparasites can now play music
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/guardian/guardian.dm [hunk 1]
```diff
@@ -84,8 +86,16 @@ GLOBAL_LIST_EMPTY(parasites) //all currently existing/living guardians
 
 /mob/living/simple_animal/hostile/guardian/Destroy()
 	GLOB.parasites -= src
+	QDEL_NULL(music_datum)
 	return ..()
 
+/mob/living/simple_animal/hostile/guardian/verb/music_interact()
+	set name = "Access Internal Synthesizer"
+	set desc = "ACcess your internal musical synthesizer"
```
Comment (Hatterhat) [inline code/modules/mob/living/simple_animal/guardian/guardian.dm]: fix the thing maybe

**Comment by TheSpaghetti** [comment]
merge this

**Comment by ghost** [comment]
yes

**Comment by MalricB** [comment]
now this is actually based

**Pull request merged**